dmscripts/notify_fw_interested_suppliers.py

```diff
@@ -1,16 +1,57 @@
+from datetime import date
+from itertools import chain
 from logging import Logger
 from typing import Optional
 from uuid import UUID, uuid4
+from warnings import warn
 
 from dmapiclient import DataAPIClient
 
 from dmutils.email import DMNotifyClient
 from dmutils.email.exceptions import EmailError
 from dmutils.email.helpers import hash_string
 from dmutils.env_helpers import get_web_url_from_stage
+from dmutils import formats
 
 
-def notify_suppliers_of_new_fw_cq_answers(
+_date_formats = (
+    "displaytimeformat",
+    "dateformat",
+    "datetimeformat",
+)
+
+
+def _maybe_time_today(key, iso_timestamp):
```

Comment:
The word `maybe` in a programming context strikes fear into my heart.

How about `_deadline_time_today(...)`? I think we can be specific, given this script is always going to be sending emails about ongoing framework applications, and any times in these are going to be deadlines.

Reply:
I think deadline is even less clear, and I partly mean for `maybe` to stand out...

Reply:
OK. Instead, could we change first part of the docstring below (`If the conditions are right`) to `If the given timestamp matches today's local date and is hour-exact`, so we're more precise there?

Reply:
ok